### PR TITLE
HTML-Forms: Mint Signup Page

### PR DESCRIPTION
**Pull request by DeliceLydia**
In this project, we created a replica of mint.com signup page using HTML and CSS.

Additional Information:
- Setup project dependencies like GitHub Actions and Linters via Microverse Linters Config.
- Defined a well structured DOM with each element properly grouped together using proper sizes and alignments.
- Used the browser's console to review mint.com's development strategy.
- Used the right input tag and attributes in the form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <div class="right signin">
+                <button type="button">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Sign In</span>
+                </button>
+            </div>
+        </div>
+        <div class="logo center-align">
+            <img src="./images/logo.png" alt="intuit logo">
+        </div>
```
Comment (SpaYco) [inline index.html]: - [x] [REQUIRED] the Intuit products should be in images and text, not a screenshot, please fix that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                </fieldset>
+                <fieldset id="input-password">
+                    <label for="password">Password</label>
+                    <img src="./images/lock_open.png" alt="padlock">
+                    <input type="password" name="password" id="password" required>
+                </fieldset>
+                <button class="form-button" type="button" name="submit">
+                    <img src="./images/lock.png" alt="sign in"> 
+                    <span>Create Account</span>
+                </button>
```
Comment (SpaYco) [inline index.html]: - [x] the button should have a `submit` type, not `button`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] your PR description can be more descriptive, please change it.

- I included more required changes below, please check them out

## Optional

- [x] make the email and password inputs required


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by andresporras3423 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
**Hello @DeliceLydia @eananti** 
Amazing job :tada: :tada: :tada:  you have met all the requirements for this project
Now you are ready to continue with your next challenge :muscle: :muscle:
Don't forget to merge with the master branch

Keep going like this  :clap: :clap:

Oscar Andrés Russi Porras

**Pull request merged**